.github/workflows/book.yml

```diff
@@ -5,8 +5,6 @@ on:
   push:
     branches:
       - "master"
-    paths:
```

Comment:
The changes to the book directory and this here should probably be in a separate commit from the "Fix examples" commit?

Reply:
Done

Reply:
This should fix the issue yes, although, I wonder if there's a way to specify a whole directory with the files in it to trigger a build.

Reply:
In that case, that's not what we want though.
Changes outside the book can break the book examples as well.

Reply:
They won't pass the CI though as the job is always run for every PR

Reply:
Ah, I see